content/body/radiogroup.php

```diff
@@ -184,7 +184,7 @@
 <?php includeStats(["isForNewBuilds" => false]); ?>
 
 <p>
-    Use when some developer before you decided making <code>&lt;div&gt;</code> tags look like radio buttons was a good use of time.
+    Use when some developer before you decided to make <code>&lt;div&gt;</code> tags look like radio buttons was a good use of time.
```

Comment:
I don't think this sentence makes sense. What about:

```suggestion
    Use when some developer before you decided that making <code>&lt;div&gt;</code> tags look like radio buttons was a good use of time.
```